discord_bot.py

```
@@ -0,0 +1,255 @@
+            capture_output=True,
+            text=True,
+            check=True
+        )
+        logger.info(f"CLI command completed successfully: {' '.join(cmd)}")
+        if result.stdout:
+            logger.debug(f"Command stdout: {result.stdout}")
+        if result.stderr:
+            logger.debug(f"Command stderr: {result.stderr}")
+            
+    except subprocess.TimeoutExpired as e:
+        logger.error(f"CLI command timed out after {timeout}s: {' '.join(cmd)}")
```

Ruff (0.12.2)

Use `logging.exception` instead of `logging.error`

Replace with `exception`

(TRY400)